build/DotNetSdkVersions.txt

```diff
@@ -1,3 +1,3 @@
 # Each line represents arguments for the .NET SDK installer script (https://learn.microsoft.com/dotnet/core/tools/dotnet-install-script)
--Channel 8.0
+-Channel 8.0.2xx
```

Comment:
This seems unrelated to enforcing code style in build.

Can you elaborate on this change?  `8.0` will use the current released version, which is currently 8.0.3xx.  Hard coding this to `8.0.2xx` will lock us into that feature band until someone bumps it.

Reply:
There's a known issue that was causing the build to fail. If we use 8.0.3xx then there is a mismatch between the compiler versions. We will need to update the value back once the latest version is available in CI. 

https://github.com/dotnet/core/blob/21da990bab4170059c6e9ed84feba7e912fa56bc/release-notes/8.0/known-issues.md#802xx-sdk-is-not-compatible-with-178-for-some-scenarios

Reply:
@dtivel is it okay to resolve this comment?